README.md

```diff
@@ -51,7 +51,7 @@ EOF
 **_WARNING:_** Until the OpenTelemetry Collector format is stable, changes may be required in the above example to remain
 compatible with the latest version of the OpenTelemetry Collector image being referenced.
 
-This will create an OpenTelemetry Collector instance named `simplest`, exposing a `jaeger-grpc` port to consume spans from your instrumented applications and exporting those spans via `logging`, which writes the spans to the console (`stdout`) of the OpenTelemetry Collector instance that receives the span.
+This will create an OpenTelemetry Collector instance named `otel`, exposing two services `otel-collector` and `otel-collector-headless`. The received spans will be exported via `logging` exporter and written to the console (`stdout`). If a gRPC protocol is used to export data to the collector use the headless service that ensures load balancing will properly work.
```

Comment:
```suggestion
This will create an OpenTelemetry Collector instance named `otel`, exposing two services `otel-collector` and `otel-collector-headless`. The received spans will be exported via `logging` exporter and written to the console (`stdout`). If a gRPC protocol is used to export data to the collector, use the headless service to ensure that load balancing will properly work.
```

Doesn't the endpoint need to be prefixed with `dns:///` for that to happen? Like `dns:///simplest-collector-headless:14250`. Just using the headless service won't ensure the load balancing will work.

Reply:
It should be enough without dns

Reply:
It's not. In fact, you should also have a balancer name set to `round_robin`, otherwise, it will use `pick_first` by default.

Reply:
The round-robin is set in the application we cannot control it here.

Reply:
Right. While I would still think that providing the actual string for them to use as an example would be useful, the text here is technically correct.